generated/nidaqmx/_install_daqmx.py

```diff
@@ -22,13 +22,16 @@
 elif sys.platform.startswith("linux"):
     import distro
 
-from nidaqmx._linux_installation_commands import _get_linux_installation_commands
-from nidaqmx._linux_installation_commands import linux_commands
+from nidaqmx._linux_installation_commands import (
+    _get_linux_installation_commands,
+    linux_commands,
+)
```

Comment:
You shouldn't need these imports for Windows builds.
```suggestion
    from nidaqmx._linux_installation_commands import (
        _get_linux_installation_commands,
        linux_commands,
    )
```